NetSuiteProject/Exercise.js

```diff
@@ -1,39 +1,34 @@
 //Add
-function add(x,y){
-	return x + y;
-};
+
 
 function addition(x,y){
-	if(typeof add (x,y) === "string"){
-	alert("This function only accepts Integers")
+	var add = (x + y);
+	if(typeof add === "string"){
+	alert("This function only accepts Numbers");
 	}
-	else {return x + y;}
+	else {return add;}
 }
 
 
 //subtract
-function subtract(x, y){
-	return (x - y);
-};
 
 function subtraction(x, y){
-	if(isNaN(subtract(x,y))  === true){
-	alert("This function only accepts Integers");
+	var subtract = (x - y);
+	if(isNaN(subtract) === true){
```

Comment:
The `=== true` portion of this conditional is valid, but it is redundant as it is the implied comparison that an `if` statement performs.  It is perfectly fine to leave it there, but know that `if(isNaN(subtract)) {` is functionally equivalent.